bondcpp/include/bondcpp/bond.hpp

```diff
@@ -1,3 +1,17 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Don't change the license.

Reply:
I have restored the license information, I have kept previous license and added new one but kept as commented, If needed it can be removed.

Reply:
And also added bondpy with a COLCON_IGNORE